CHANGELOG.md

```diff
@@ -12,3 +12,4 @@
 - Added new metric `beacon_earliest_available_slot`.
 
 ### Bug Fixes
+- Teku may crash when shutting down
```

Comment:
hrm thats not really a 'bug fix' statement
We'll want to make that more release note like.